src/jloptions.c

```diff
@@ -129,6 +130,8 @@ static const char opts[]  =
     // instrumentation options
     " --code-coverage={none|user|all}, --code-coverage\n"
     "                           Count executions of source lines (omitting setting is equivalent to \"user\")\n"
+    " --code-coverage=tracefile.info\n"
+    "                           Append coverage information to the LCOV tracefile (filename supports format tokens).\n"
```

Comment:
I don't really want to point people to the source file from here, but I'm not sure where to include the list either. I've been meaning to add a additional `--help-verbose` mode flag that goes all out with this and shows the whole man page (maybe somehow running the pager too?), perhaps it can wait for that?